Makefile

```diff
@@ -7,13 +7,23 @@ help:
 start: ## Spin up the containers and run the app UI in watch mode
 	docker compose up -d
 
-stop: ## Shut down the containers
+stop: ## Stop the containers
```

Comment:
Why would you need to stop and not remove the containers in a dev env?

Reply:
It makes sense when you have a large database with migrations and seeders already running at some point and you just want to stop your environment to continue working the next day.